rollup.config.mjs

```diff
@@ -4,53 +4,65 @@ import typescript from "@rollup/plugin-typescript";
 import dts from "rollup-plugin-dts";
 import postcss from "rollup-plugin-postcss";
 import image from '@rollup/plugin-image';
-import json from '@rollup/plugin-json'
+import json from '@rollup/plugin-json';
 import terser from "@rollup/plugin-terser";
 import peerDepsExternal from 'rollup-plugin-peer-deps-external';
 import analyze from 'rollup-plugin-analyzer';
 
 const limitBytes = 6.1e6;
 
 const onAnalysis = ({ bundleSize }) => {
-	if (bundleSize < limitBytes) return
-	console.log(`Bundle size exceeds ${limitBytes/1024} kb: ${bundleSize/1024} kb`)
-	return process.exit(1)
-}
+	if (bundleSize < limitBytes) return;
+	console.log(`Bundle size exceeds ${limitBytes / 1024} kb: ${bundleSize / 1024} kb`);
```

Comment:
Anybody know what this actually counts? I only left this on for the browser bundle which is 2.3MB but it still says it's close to 6 MB. I have no idea where the 6 is coming from here.

![image](https://github.com/user-attachments/assets/ee57f85f-cb33-454d-865e-bf1e2eb2032b)

Reply:
It sounds like the plugin we are using is more about examining relative module size and bloat than delivery size:  https://github.com/doesdev/rollup-plugin-analyzer?tab=readme-ov-file#faq

I believe the bundle size we are seeing is the pre-minified bundle size and it is computed by summing the size of all the included modules:
```
const onAnalysis = (results) => {
	let computedSize = results.modules.reduce((total,module) => total + module.size, 0);
	console.log(`Bundle size: ${results.bundleSize / 1024} kb, computed: ${computedSize / 1024} kb`);
	process.exit(1);
};
```

If you remove the terser plugin, the browser output file sizes are pretty close to the reported bundle size.

Reply:
Gotcha. Cool. Thanks for looking into it.